compile_test.go

```diff
@@ -69,6 +72,7 @@ func testFile(t *testutil.T, file string) {
 		f, err := parser.ParseFile(fset, go_file, go_source, parser.AllErrors)
 		t.AssertNoError(err)
 
+		t.Logf("[%s] checking imports...", file)
```

Comment:
a better line for this is "checking compiles..." because it's not just checking the imports, it's doing full type checking.